mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -77,7 +86,7 @@ object AutoImports:
     new AutoImportsGenerator(
       pos,
       importPos,
-      indexedContext.importContext,
```

Comment:
We shouldn't use `.importContext` in whole `AutoImportsGenerator` because, for example, https://github.com/scalameta/metals/blob/cab18fe05646abaa310d0067e3097298044197f1/mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala#L157 should look up symbol in the context where auto-completion has invoked, instead of import context.
The only place we should use `.importContext` is https://github.com/scalameta/metals/pull/3897/files#diff-ea3d329f887bd07d2542b91d54a4a2527f468ae48176d004f7dbfe32a3adf3edR195